translationCoreApps/ScripturePane
# Feature - #2843 - Makes it obvious when Greek text not available

#### This pull request addresses:

Address issue https://github.com/unfoldingWord-dev/translationCore/issues/2843. It actually addresses any verse missing from any Bible translation. 

**NOTE:** This also moves the PLACE HOLDER (i.e. warning text) constant into the Verse component rather than components that render Verse to set the text, but rather just pass `null` and Verse.js checks if it is null and displays the warning. This is because 1) two places were setting the place holder and 2) Verse needs to know it is a place holder so it can style it correctly.


#### How to test this pull request:

1) Load a project of a book other than Titus. Go into tW tool and you should see for any verse the Greek column says "[WARNING: This Bible version does not include text for this reference.]"

2) Remove/rename the the `~/translationCore/resources/en/bibles/ulb/v11/tit` directory from your home directory. Reload the tC app, load a Titus project, go into tW tool for that project, load the resource in the Scripture Pane if not already, and it also should give you the warning.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/translationcoreapps/scripturepane/73)
